extensions/GlobalMessages/tests/GlobalMessages0.test.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+require '/usr/wikia/source/wiki/maintenance/Maintenance.php';
+
+error_reporting(E_ALL);
+ini_set('display_errors', 1);
+ini_set('log_errors', 0);
+ini_set('html_errors', 0);
+
+class GlobalMessages0Test extends Maintenance {
+
+	public function execute() {
+
+		$this->dataProvider = require '/usr/wikia/source/wiki/extensions/GlobalMessages/tests/GlobalMessages0.data.php';
+
+//		$this->docProvider = require '/usr/wikia/source/wiki/extensions/GlobalMessages/tests/qqq.raw.php';
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this commented out code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS125)